### ISSUE TITLE
 Show block deps

### ISSUE THREAD

**Issue by tdroxler** [alephium/explorer#93]
In the returned block entity we have a `deps` field, which is a list of block hashes, could be nice to show it in the explorer as the data is already there.

**Comment by lf94** [comment]
The ExplorerClient doesn't expose `deps` in any block structures from what I'm seeing; do we need to export something to `@alephium/sdk` ?

**Comment by mvaivre** [comment]
@tdroxler Candide question: it seems like this info wouldn't be useful for the average user. Would you say that this could be useful for you, or for our tech savvy devs?

**Comment by tdroxler** [comment]
I don't think it would be super useful even for some techy guy, maybe only 1-2 guys :sweat_smile: The idea is more to show the data we have and the "mechanism" of `BlockFlow`
It could be put in a "Show more info" , the same way etherscan is doing 
![image](https://user-images.githubusercontent.com/2979182/167372040-d2d6dd84-9b60-4408-b02d-ae88a45b55da.png)
 
But it's really not urgent, I created the issue so we don't forget we have those data somewhere.
